app/build.gradle

```diff
@@ -90,6 +91,7 @@ dependencies {
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation 'com.google.firebase:firebase-crashlytics:17.2.2'
```

Comment:
I guess you should drop versions from all other firebase dependencies as well since you are using `firebase-bom`.